### ISSUE TITLE
Add new icons

### ISSUE THREAD

**Issue by miketaylr** [webcompat/webcompat-reporter-extensions#80]
From redesign.

**Comment by miketaylr** [comment]
But let's wait for the redesign to land, so we're not confusing branding or whatever. 

I suppose https://github.com/webcompat/webcompat.com/issues/2017 is as good as any issue to track for that!

**Comment by miketaylr** [comment]
> But let's wait for the redesign to land, so we're not confusing branding or whatever.

Actually, someone should be able to work on this now, and we'll just merge when ready!

**Comment by miketaylr** [comment]
> @miketaylr there is this icons folder under img folder, is this the one?

I don't think that's the one, we want to use the favicon-ish ones, like https://github.com/webcompat/webcompat.com/blob/refactor/webcompat/static/favicon/apple-touch-icon-180x180.png. But possibly there's a source file that @zoepage can point us at.

**Comment by zoepage** [comment]
Is this what you are looking for?
 https://github.com/webcompat/design/tree/master/archive/logos

**Comment by ncarbon** [comment]
@miketaylr @zoepage Sorry if this sounds like an obvious question but, how do I know their sizes? (default sizes: 32, 64, 128, etc) 😅

**Comment by zoepage** [comment]
@ncarbon that is actually a very good question! 😅
Guess we need to wait until @miketaylr comes back from vacation. :)

**Comment by miketaylr** [comment]
> Sorry if this sounds like an obvious question but, how do I know their sizes? (default sizes: 32, 64, 128, etc)

@ncarbon yep! @zoepage is correct, we want to have a 32x32px (and... 48px, 64px, 128px) icon to replace the existing ones in https://github.com/webcompat/webcompat-reporter-extensions/tree/master/shared.

I would recommend downloading sketch: https://www.sketchapp.com/ (there's a 30 day trial) and opening the sketch file in that directory, and learn how to export them at the desired sizes. Let me know if you get stuck!

(sorry to be so delayed in responding, have been on PTO)

**Comment by ncarbon** [comment]
@miketaylr Is "Logo Black on Gold" the one we want for the browser extensions?

**Comment by miketaylr** [comment]
@ncarbon yep!
